### PR TITLE
LAPP-26

### PR DESCRIPTION
**Pull request by cgalan-applaudostudios**
Replacing AsyncFlushHandler with AsyncIntervalFlushHandler on target Net35

this depends on #117

### CODE REVIEW

File: Analytics.Portable/Analytics.Portable.csproj [hunk 1]
```diff
@@ -41,15 +41,5 @@
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
   </ItemGroup>
-  <ItemGroup>
-    <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
-      <_Parameter1>Test.UniversalApp</_Parameter1>
```
Comment (lubird) [inline Analytics.Portable/Analytics.Portable.csproj:46]: Why do we remove these test projects?

Comment (cgalan-applaudostudios) [reply Analytics.Portable/Analytics.Portable.csproj:46]: These lines were to make visible the internal classes to the Test.UniversalApp project, but the PCL (portable class library) project doesn't support this configuration. I had to add it programmatically in the file _AssemblyConfiguration.cs. But at that moment I forgot to remove these lines. By the way, PCL projects are considered [deprecated](https://docs.microsoft.com/en-us/xamarin/cross-platform/app-fundamentals/pcl?tabs=macos).

**Comment by lubird** [comment]
What does this change have from a user perspective? Will it enable async operation on Net35?

**Comment by cgalan-applaudostudios** [comment]
From the user perspective, this will enable the use of multiple threads. And also with this change there will be only one async flush manager and the code base will be more simple.

**Pull request merged**